### PR TITLE
Add wsmancmd --file support 

### PR DESCRIPTION
**Pull request by dragoshsch**
Add wsmancmd.py --file support for powershell scripts (doesn't support comments in the PS scripts right now) and update the wsmancmd linux binary.

### CODE REVIEW

File: DCOS/utils/wsmancmd.py [hunk 1]
```diff
@@ -29,11 +29,11 @@
...
-    print ("%s [-U <url>] [-H <host>] [-P <port>] [-s] "
-           "[-a <basic|kerberos|certificate>] "
-           "[-u <username>] [-p <password>] "
-           "[-c <client_cert_pem> -k <client_cert_cert_key_pem>] "
-           "<cmd> [cmd_args]" % sys.argv[0])
+    print("%s [-U <url>] [-H <host>] [-P <port>] [-s] "
+          "[-a <basic|kerberos|certificate>] "
+          "[-u <username>] [-p <password>] "
+          "[-c <client_cert_pem> -k <client_cert_cert_key_pem>] "
+          "--file <powershell_script> --powershell <cmd> [cmd_args]" % sys.argv[0])
```
Comment (ionutbalutoiu) [inline DCOS/utils/wsmancmd.py]: I would say we leave the `--file` generic. If somebody wants to execute a remote script (without PowerShell) from file, he can do so.

Comment (ionutbalutoiu) [inline DCOS/utils/wsmancmd.py]: I would say we leave the `--file` generic. If somebody wants to execute a remote script (without PowerShell) from file, he can do so.

File: DCOS/utils/wsmancmd.py [hunk 2]
```diff
@@ -76,7 +77,14 @@ def parse_args():
                 cert_key_pem = arg
             elif opt == "--powershell":
                 is_powershell_cmd = True
-        cmd = args
+            elif opt == "--file":
+                is_powershell_script = True
```
Comment (ionutbalutoiu) [inline DCOS/utils/wsmancmd.py]: Change variable from `is_powershell_script` to `is_script`

File: DCOS/utils/wsmancmd.py [hunk 3]
```diff
@@ -76,7 +77,14 @@ def parse_args():
...
+        if is_powershell_cmd and not is_powershell_script:
```
Comment (ionutbalutoiu) [inline DCOS/utils/wsmancmd.py]: Let's make a simple check here:

```
if is_script:
    with open(args[0], 'r') as script_file:
        content = script_file.read()
    cmd = content.replace('\n', '; ')
else:
    cmd = args
```

File: DCOS/utils/wsmancmd.py [hunk 4]
```diff
@@ -91,7 +99,7 @@ def parse_args():
     except getopt.GetoptError:
         print_usage()
     return (args_ok, url, host, use_ssl, port, auth, username, password,
-            cert_pem, cert_key_pem, cmd, is_powershell_cmd)
+            cert_pem, cert_key_pem, cmd, is_powershell_cmd, is_powershell_script)
```
Comment (ionutbalutoiu) [inline DCOS/utils/wsmancmd.py]: We don't really need to pass the `is_script` variable going forward.

File: DCOS/utils/wsmancmd.py [hunk 5]
```diff
@@ -135,14 +143,19 @@ def get_url(url, host, use_ssl, port):
...
-     cert_pem, cert_key_pem, cmd, is_powershell_cmd) = parse_args()
+     cert_pem, cert_key_pem, cmd, is_powershell_cmd, is_powershell_script) = parse_args()
     if args_ok:
         url = get_url(url, host, use_ssl, port)
         if is_powershell_cmd:
             cmd = ["powershell.exe",
                    "-ExecutionPolicy", "RemoteSigned", "-NonInteractive",
                    "-EncodedCommand",
                    base64.b64encode((" ".join(cmd)).encode("utf-16-le"))]
+        if is_powershell_script:
```
Comment (ionutbalutoiu) [inline DCOS/utils/wsmancmd.py]: The above code works anyway. Of course, if we passed `--file`, then it doesn't join anything, but it won't fail either.

I would recommend that we remove this. Makes the code cleaner to have one less `if`.

File: DCOS/utils/wsmancmd.py (hunk 3 above)
Comment (ionutbalutoiu) [reply DCOS/utils/wsmancmd.py]: Looks like we need another char for one-line commands in `CMD`.

The equivalent of `;` in `CMD` is `&`.

**Comment by dragoshsch** [comment]
Added all the requested changes. The script now works for both cmd and powershell commands, from file, or given as arguments.

**Comment by dragoshsch** [comment]
Cleaned up if statements and added 'cmd /c' runner for non-powershell commands.

**Pull request merged**